### DISCUSSION TITLE
Crash after unplugging the secondary displayer.

### DISCUSSION THREAD

**Discussion by StLeoX** [flameshot-org/flameshot#3293] [Q&A]
It works well if I use two displayers. But if I unplug the secondary displayer and use the origin displayer of my laptop. Then it will crash after `flameshot gui`.
The crash is, the screen will flash, and the opening window of other apps will resize.

my env:
```
flameshot --version
Flameshot v12.1.0 ()
Compiled with Qt 5.11.3

OS: Linux Mint 21 x86_64, with cinnamon
```

Anyone knows the reason?

**Comment by mmahmoudian** [comment]
The reason is that Flameshot does not constantly refresh the monitor status. The solution for now is to kill Flameshot and start it again after monitor change has happened.

**Comment by StLeoX** [comment]
Maybe not.
Between the good and bad status, I've reboot, or even reinstall flameshot on my laptop.

**Comment by mmahmoudian** [comment]
Try this and tell me what you see:
1. Disconnect all external monitors
2. Run `pkill --full flameshot` in terminal 
3. Run `flameshot` in terminal (and keep this terminal open)
4. In another terminal try `flamrshot gui` (we call this terminal 2)
5. Is everything working normally?
6. Close the screenshot of Flameshot
7. Connect your external display
8. Run `flameshot gui` again in the terminal 2
9. Is everything working correctly?
10. Close the screenshot of Flameshot
11. Repeat from step2 but this time disconnect the external monitor on step7
